load_test/lib/load_test/communication/messages.pb.ex

```diff
@@ -105,8 +106,8 @@ defmodule LoadTest.Communication.Proto.LobbyEventType do
   field(:TYPE_UNSPECIFIED, 0)
   field(:CONNECTED, 1)
   field(:PLAYER_ADDED, 2)
-  field(:GAME_STARTED, 3)
-  field(:START_GAME, 4)
+  field(:START_GAME, 3)
```

Comment:
do we still use `START_GAME`?

Reply:
You're right! this message was not being used anywhere, deleted

Reply:
You're right! this message was not being used anywhere, deleted